Gateway/Request/DeviceDataBuilder.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © 2013-2017 Magento, Inc. All rights reserved.
```

Comment:
Remove date from license (possibly remove entirely?)

Reply:
Yea, was never too sure on these. Obviously lots of them in Braintree codebase due to forking the original module.